app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_line = command.split()
+    command_name, source_file, copy_file_name = command_line
+    is_valid_file_names = source_file != copy_file_name
+    is_valid_command = command_name == "cp"
+    is_valid_count_of_parameters = len(command_line) == 3
+    if all(is_valid_command,
+           is_valid_file_names,
+           is_valid_count_of_parameters
+           ):
```

Comment:
The use of `all()` here is incorrect. `all()` expects an iterable, but you are passing multiple arguments directly. You should pass a list or tuple of conditions to `all()`, like `all([is_valid_command, is_valid_file_names, is_valid_count_of_parameters])`.